src/commands/org/list/metadata-types.ts

```diff
@@ -68,7 +68,18 @@ export class ListMetadataTypes extends SfCommand<DescribeMetadataResult> {
       await fs.promises.writeFile(flags['output-file'], JSON.stringify(describeResult, null, 2));
       this.logSuccess(`Wrote result file to ${flags['output-file']}.`);
     } else {
-      this.styledJSON(describeResult);
+      this.table(
+        describeResult.metadataObjects,
+        Object.fromEntries(Object.keys(describeResult.metadataObjects[0]).map((k) => [k, { header: k }])),
```

Comment:
the type says that suffix is potentially undefined.  If that's true (?), and the first item didn't have a suffix, then the suffix column wouldn't exist even if other items did have one.